deps/build.jl

```diff
@@ -14,7 +15,7 @@ short_version = "1.8.1"
 zipname = "lz4_v$(replace(short_version, "." => "_"))_win$(Sys.WORD_SIZE).zip"
 
 suffix = "so.$short_version"
-if Sys.is_apple()
+if isapple()
```

Comment:
Prefer using `Sys.isapple()` as this is the standard usage on 1.0. You'll just need to add `import Compat: Sys` at the top. The advantage of this approach is you just need to remove the import when you drop Julia 0.6.